frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -1199,8 +1199,32 @@ export default ({
       if (this.dndState.isActive) {
         this.dndState.isActive = false
 
-        e?.dataTransfer.files?.length &&
-          this.$refs.sendArea.fileAttachmentHandler(e?.dataTransfer.files, true)
+        const items = e.dataTransfer.items
+
+        if (items?.length) {
+          // 'drag-end' event of the browsers detects a folder too and turns it into a File() instance, which is an unwanted behaviour.
+          // So filtering out directories here by using .webkitGetAsEntry() method provided via DataTransferItem API.
+          // (Reference: https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem/webkitGetAsEntry)
```

Comment:
`webkitGetAsEntry` method here seems to have a reliable level of supports for desktop browsers:
https://caniuse.com/mdn-api_datatransferitem_webkitgetasentry